prime-router/src/main/kotlin/history/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -38,19 +42,15 @@ interface SubmissionAccess {
         klass: Class<T>
     ): List<T>
 
-    fun <T, P, U> fetchAction(
+    fun <T> fetchAction(
         sendingOrg: String,
         submissionId: Long,
-        klass: Class<T>,
-        reportsKlass: Class<P>,
-        logsKlass: Class<U>,
+        klass: Class<T>
     ): T?
 
-    fun <T, P, U> fetchRelatedActions(
+    fun <T> fetchRelatedActions(
```

Comment:
Add kdoc describing what this is doing. Understandable this is not code you originally wrote, but please document it in this PRc

Reply:
done